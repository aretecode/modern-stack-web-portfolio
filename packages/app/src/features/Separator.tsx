import styled from 'styled-components'

export const StyledSeparator = styled.div.attrs({
  role: 'separator',
  children: '-',
})`
  color: var(--color-text-body);
  display: inline-flex;
  padding: 0 0.5rem;
`
