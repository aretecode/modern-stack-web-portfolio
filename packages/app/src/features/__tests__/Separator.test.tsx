import * as React from 'react'
import { render } from 'react-testing-library'
import { StyledSeparator } from '../Separator'

describe('Separator', () => {
  it('should match snapshot', () => {
    const { container } = render(<StyledSeparator />)
    expect(container).toMatchSnapshot()
  })
})
