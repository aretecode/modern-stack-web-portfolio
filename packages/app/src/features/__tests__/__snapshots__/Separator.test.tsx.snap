// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator should match snapshot 1`] = `
.c0 {
  color: var(--color-text-body);
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 0 0.5rem;
}

<div>
  <div
    class="c0"
    role="separator"
  >
    -
  </div>
</div>
`;
